pkgs/development/python-modules/readmdict/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+
+, poetry-core
+, python-lzo
+}:
+
+buildPythonPackage rec {
+  pname = "readmdict";
+  version = "0.1.1";
+  pyproject = true;
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-RVVvXOR2tnPN+ab5Ak7YHLbgGq7AE5Es7cFJqjvTvrY=";
+  };
+
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+      --replace "poetry>=0.12" "poetry-core" \
+      --replace "poetry.masonry" "poetry.core.masonry"
+  '';
+
+  nativeBuildInputs = [
+    poetry-core
+  ];
+
+  propagatedBuildInputs = [
+    python-lzo
+  ];
+
```

Comment:
```suggestion

  nativeCheckInputs = [
    pytestCheckHook
  ]

```

Reply:
The runner can't find any:

```
python3.10-readmdict> Executing pytestCheckPhase
python3.10-readmdict> ============================= test session starts ==============================
python3.10-readmdict> platform linux -- Python 3.10.12, pytest-7.4.0, pluggy-1.2.0
python3.10-readmdict> rootdir: /build/readmdict-0.1.1
python3.10-readmdict> collected 0 items
python3.10-readmdict> ============================ no tests ran in 0.00s =============================
python3.10-readmdict> /nix/store/vr6wwdxkmyy44sg0gwxi10b8fc5zhwz0-stdenv-linux/setup: line 1596: pop_var_context: head of shell_variables not a function context
error: builder for '/nix/store/vascw48f6bm1r0nm8dvjyxi9wkjzh7h6-python3.10-readmdict-0.1.1.drv' failed with exit code 5;
       last 10 log lines:
       > Check whether the following modules can be imported: readmdict
       > pytestCheckPhase
       > Executing pytestCheckPhase
       > ============================= test session starts ==============================
       > platform linux -- Python 3.10.12, pytest-7.4.0, pluggy-1.2.0
       > rootdir: /build/readmdict-0.1.1
       > collected 0 items
       >
       > ============================ no tests ran in 0.00s =============================
       > /nix/store/vr6wwdxkmyy44sg0gwxi10b8fc5zhwz0-stdenv-linux/setup: line 1596: pop_var_context: head of shell_variables not a function context
       For full logs, run 'nix log /nix/store/vascw48f6bm1r0nm8dvjyxi9wkjzh7h6-python3.10-readmdict-0.1.1.drv'.
```

[There is one](https://github.com/ffreemt/readmdict/blob/967d91427174a38104d03354bec58345cf78d7f4/tests/test_version.py), however, but it just check the version.

Reply:
Checked the archive from PyPI, there aren't any tests packaged.

Reply:
The test might be trivial, but upstream may chose to add more in the future. Consider using `fetchFromGitHub`